frontend/src/Button.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import './index.css';
+
+class Button extends Component {
+    render(){
+        return (
+            <div>
+                <button className="btn">{this.props.number}</button>
```

Comment:
You forgot to add functionality so this can redirect into a page. Just mock the onClick function at the very least.